[BUG] Quantized Initialization will enter compilation error

**Describe the bug**
Repect to https://github.com/deepspeedai/DeepSpeedExamples/tree/master/inference/huggingface/zero_inference#quantized-initialization
I got:

**To Reproduce**

```
benchmark, prompt_len = 512, execute_gen_len = 32, input_ids.shape = torch.Size([80, 512])
Using /home/wzy/.cache/torch_extensions/py313_cu128 as PyTorch extensions root...
Detected CUDA files, patching ldflags
Emitting ninja build file /home/wzy/.cache/torch_extensions/py313_cu128/quantizer/build.ninja...
/usr/lib/python3.13/site-packages/torch/utils/cpp_extension.py:2356: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation.
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
Building extension module quantizer...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
[1/3] /opt/cuda/bin/nvcc --generate-dependencies-with-compile --dependency-output fake_quantizer.cuda.o.d -DTORCH_EXTENSION_NAME=quantizer -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1019\" -I/usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes -I/usr/lib/python3.13/site-packages/torch/include -I/usr/lib/python3.13/site-packages/torch/include/torch/csrc/api/include -I/opt/cuda/include -I/usr/include/python3.13 -D_GLIBCXX_USE_CXX11_ABI=1 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_75,code=compute_75 -gencode=arch=compute_75,code=sm_75 --compiler-options '-fPIC' -O3 --use_fast_math -std=c++20 -U__CUDA_NO_HALF_OPERATORS__ -U__CUDA_NO_HALF_CONVERSIONS__ -U__CUDA_NO_HALF2_OPERATORS__ --threads=8 -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_75,code=compute_75 -c /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/fake_quantizer.cu -o fake_quantizer.cuda.o
FAILED: fake_quantizer.cuda.o
/opt/cuda/bin/nvcc --generate-dependencies-with-compile --dependency-output fake_quantizer.cuda.o.d -DTORCH_EXTENSION_NAME=quantizer -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1019\" -I/usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes -I/usr/lib/python3.13/site-packages/torch/include -I/usr/lib/python3.13/site-packages/torch/include/torch/csrc/api/include -I/opt/cuda/include -I/usr/include/python3.13 -D_GLIBCXX_USE_CXX11_ABI=1 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_75,code=compute_75 -gencode=arch=compute_75,code=sm_75 --compiler-options '-fPIC' -O3 --use_fast_math -std=c++20 -U__CUDA_NO_HALF_OPERATORS__ -U__CUDA_NO_HALF_CONVERSIONS__ -U__CUDA_NO_HALF2_OPERATORS__ --threads=8 -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_75,code=compute_75 -c /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/fake_quantizer.cu -o fake_quantizer.cuda.o
In file included from /usr/include/c10/util/logging_is_google_glog.h:50,
                 from /usr/include/c10/util/Logging.h:27,
                 from /usr/include/ATen/cuda/CUDAContext.h:8,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes/context.h:8,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes/custom_cuda_layers.h:16,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/fake_quantizer.cu:7:
/usr/include/glog/logging.h:60:4: error: #error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
   60 | #  error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:63:
/usr/include/glog/flags.h:45:4: error: #error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
   45 | #  error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:77:
/usr/include/glog/log_severity.h:38:4: error: #error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
   38 | #  error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:78:
/usr/include/glog/vlog_is_on.h:71:4: error: #error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
   71 | #  error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/c10/util/logging_is_google_glog.h:50,
                 from /usr/include/c10/util/Logging.h:27,
                 from /usr/include/ATen/cuda/CUDAContext.h:8,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes/context.h:8,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes/custom_cuda_layers.h:16,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/fake_quantizer.cu:7:
/usr/include/glog/logging.h:60:4: error: #error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
   60 | #  error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:63:
/usr/include/glog/flags.h:45:4: error: #error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
   45 | #  error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:77:
/usr/include/glog/log_severity.h:38:4: error: #error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
   38 | #  error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:78:
/usr/include/glog/vlog_is_on.h:71:4: error: #error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
   71 | #  error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
fatal   : Could not open input file /tmp/tmpxft_0002791f_00000000-7_fake_quantizer.cpp1.ii
[2/3] c++ -MMD -MF pt_binding.o.d -DTORCH_EXTENSION_NAME=quantizer -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1019\" -I/usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes -I/usr/lib/python3.13/site-packages/torch/include -I/usr/lib/python3.13/site-packages/torch/include/torch/csrc/api/include -I/opt/cuda/include -I/usr/include/python3.13 -D_GLIBCXX_USE_CXX11_ABI=1 -fPIC -std=c++17 -O3 -std=c++17 -g -Wno-reorder -c /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/pt_binding.cpp -o pt_binding.o
FAILED: pt_binding.o
c++ -MMD -MF pt_binding.o.d -DTORCH_EXTENSION_NAME=quantizer -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1019\" -I/usr/lib/python3.13/site-packages/deepspeed/ops/csrc/includes -I/usr/lib/python3.13/site-packages/torch/include -I/usr/lib/python3.13/site-packages/torch/include/torch/csrc/api/include -I/opt/cuda/include -I/usr/include/python3.13 -D_GLIBCXX_USE_CXX11_ABI=1 -fPIC -std=c++17 -O3 -std=c++17 -g -Wno-reorder -c /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/pt_binding.cpp -o pt_binding.o
In file included from /usr/include/c10/util/logging_is_google_glog.h:50,
                 from /usr/include/c10/util/Logging.h:27,
                 from /usr/include/ATen/cuda/CUDAContext.h:8,
                 from /usr/lib/python3.13/site-packages/deepspeed/ops/csrc/quantization/pt_binding.cpp:6:
/usr/include/glog/logging.h:60:4: error: #error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
   60 | #  error <glog/logging.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:63:
/usr/include/glog/flags.h:45:4: error: #error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
   45 | #  error <glog/flags.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:77:
/usr/include/glog/log_severity.h:38:4: error: #error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
   38 | #  error <glog/log_severity.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
In file included from /usr/include/glog/logging.h:78:
/usr/include/glog/vlog_is_on.h:71:4: error: #error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
   71 | #  error <glog/vlog_is_on.h> was not included correctly. See the documentation for how to consume the library.
      |    ^~~~~
/usr/include/glog/flags.h:105:1: error: ‘GLOG_EXPORT’ does not name a type
  105 | DECLARE_int32(logemaillevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:105:1: error: ‘FLAGS_logemaillevel’ has not been declared in ‘fLI’
  105 | DECLARE_int32(logemaillevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:106:1: error: ‘GLOG_EXPORT’ does not name a type
  106 | DECLARE_int32(logcleansecs);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:106:1: error: ‘FLAGS_logcleansecs’ has not been declared in ‘fLI’
  106 | DECLARE_int32(logcleansecs);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:109:1: error: ‘GLOG_EXPORT’ does not name a type
  109 | DECLARE_bool(drop_log_memory);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:109:1: error: ‘FLAGS_drop_log_memory’ has not been declared in ‘fLB’
  109 | DECLARE_bool(drop_log_memory);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:111:1: error: ‘GLOG_EXPORT’ does not name a type
  111 | DECLARE_string(alsologtoemail);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:111:1: error: ‘FLAGS_alsologtoemail’ has not been declared in ‘fLS’
  111 | DECLARE_string(alsologtoemail);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:112:1: error: ‘GLOG_EXPORT’ does not name a type
  112 | DECLARE_string(log_backtrace_at);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:112:1: error: ‘FLAGS_log_backtrace_at’ has not been declared in ‘fLS’
  112 | DECLARE_string(log_backtrace_at);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:115:1: error: ‘GLOG_EXPORT’ does not name a type
  115 | DECLARE_bool(timestamp_in_logfile_name);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:115:1: error: ‘FLAGS_timestamp_in_logfile_name’ has not been declared in ‘fLB’
  115 | DECLARE_bool(timestamp_in_logfile_name);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:118:1: error: ‘GLOG_EXPORT’ does not name a type
  118 | DECLARE_bool(logtostdout);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:118:1: error: ‘FLAGS_logtostdout’ has not been declared in ‘fLB’
  118 | DECLARE_bool(logtostdout);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:121:1: error: ‘GLOG_EXPORT’ does not name a type
  121 | DECLARE_bool(colorlogtostdout);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:121:1: error: ‘FLAGS_colorlogtostdout’ has not been declared in ‘fLB’
  121 | DECLARE_bool(colorlogtostdout);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:124:1: error: ‘GLOG_EXPORT’ does not name a type
  124 | DECLARE_bool(logtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:124:1: error: ‘FLAGS_logtostderr’ has not been declared in ‘fLB’
  124 | DECLARE_bool(logtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:127:1: error: ‘GLOG_EXPORT’ does not name a type
  127 | DECLARE_bool(alsologtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:127:1: error: ‘FLAGS_alsologtostderr’ has not been declared in ‘fLB’
  127 | DECLARE_bool(alsologtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:130:1: error: ‘GLOG_EXPORT’ does not name a type
  130 | DECLARE_bool(colorlogtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:130:1: error: ‘FLAGS_colorlogtostderr’ has not been declared in ‘fLB’
  130 | DECLARE_bool(colorlogtostderr);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:134:1: error: ‘GLOG_EXPORT’ does not name a type
  134 | DECLARE_int32(stderrthreshold);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:134:1: error: ‘FLAGS_stderrthreshold’ has not been declared in ‘fLI’
  134 | DECLARE_int32(stderrthreshold);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:137:1: error: ‘GLOG_EXPORT’ does not name a type
  137 | DECLARE_bool(log_file_header);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:137:1: error: ‘FLAGS_log_file_header’ has not been declared in ‘fLB’
  137 | DECLARE_bool(log_file_header);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:140:1: error: ‘GLOG_EXPORT’ does not name a type
  140 | DECLARE_bool(log_prefix);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:140:1: error: ‘FLAGS_log_prefix’ has not been declared in ‘fLB’
  140 | DECLARE_bool(log_prefix);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:143:1: error: ‘GLOG_EXPORT’ does not name a type
  143 | DECLARE_bool(log_year_in_prefix);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:143:1: error: ‘FLAGS_log_year_in_prefix’ has not been declared in ‘fLB’
  143 | DECLARE_bool(log_year_in_prefix);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:147:1: error: ‘GLOG_EXPORT’ does not name a type
  147 | DECLARE_int32(logbuflevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:147:1: error: ‘FLAGS_logbuflevel’ has not been declared in ‘fLI’
  147 | DECLARE_int32(logbuflevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:150:1: error: ‘GLOG_EXPORT’ does not name a type
  150 | DECLARE_int32(logbufsecs);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:150:1: error: ‘FLAGS_logbufsecs’ has not been declared in ‘fLI’
  150 | DECLARE_int32(logbufsecs);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:154:1: error: ‘GLOG_EXPORT’ does not name a type
  154 | DECLARE_int32(minloglevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:154:1: error: ‘FLAGS_minloglevel’ has not been declared in ‘fLI’
  154 | DECLARE_int32(minloglevel);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:158:1: error: ‘GLOG_EXPORT’ does not name a type
  158 | DECLARE_string(log_dir);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:158:1: error: ‘FLAGS_log_dir’ has not been declared in ‘fLS’
  158 | DECLARE_string(log_dir);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:161:1: error: ‘GLOG_EXPORT’ does not name a type
  161 | DECLARE_int32(logfile_mode);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:161:1: error: ‘FLAGS_logfile_mode’ has not been declared in ‘fLI’
  161 | DECLARE_int32(logfile_mode);
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:165:1: error: ‘GLOG_EXPORT’ does not name a type
  165 | DECLARE_string(log_link);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:165:1: error: ‘FLAGS_log_link’ has not been declared in ‘fLS’
  165 | DECLARE_string(log_link);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:167:1: error: ‘GLOG_EXPORT’ does not name a type
  167 | DECLARE_int32(v);  // in vlog_is_on.cc
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:167:1: error: ‘FLAGS_v’ has not been declared in ‘fLI’
  167 | DECLARE_int32(v);  // in vlog_is_on.cc
      | ^~~~~~~~~~~~~
/usr/include/glog/flags.h:169:1: error: ‘GLOG_EXPORT’ does not name a type
  169 | DECLARE_string(vmodule);  // also in vlog_is_on.cc
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:169:1: error: ‘FLAGS_vmodule’ has not been declared in ‘fLS’
  169 | DECLARE_string(vmodule);  // also in vlog_is_on.cc
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:172:1: error: ‘GLOG_EXPORT’ does not name a type
  172 | DECLARE_uint32(max_log_size);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:172:1: error: ‘FLAGS_max_log_size’ has not been declared in ‘fLU’
  172 | DECLARE_uint32(max_log_size);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:175:1: error: ‘GLOG_EXPORT’ does not name a type
  175 | DECLARE_bool(stop_logging_if_full_disk);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:175:1: error: ‘FLAGS_stop_logging_if_full_disk’ has not been declared in ‘fLB’
  175 | DECLARE_bool(stop_logging_if_full_disk);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:178:1: error: ‘GLOG_EXPORT’ does not name a type
  178 | DECLARE_bool(log_utc_time);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:178:1: error: ‘FLAGS_log_utc_time’ has not been declared in ‘fLB’
  178 | DECLARE_bool(log_utc_time);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:181:1: error: ‘GLOG_EXPORT’ does not name a type
  181 | DECLARE_string(logmailer);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:181:1: error: ‘FLAGS_logmailer’ has not been declared in ‘fLS’
  181 | DECLARE_string(logmailer);
      | ^~~~~~~~~~~~~~
/usr/include/glog/flags.h:183:1: error: ‘GLOG_EXPORT’ does not name a type
  183 | DECLARE_bool(symbolize_stacktrace);
      | ^~~~~~~~~~~~
/usr/include/glog/flags.h:183:1: error: ‘FLAGS_symbolize_stacktrace’ has not been declared in ‘fLB’
  183 | DECLARE_bool(symbolize_stacktrace);
      | ^~~~~~~~~~~~
/usr/include/glog/vlog_is_on.h:111:8: error: ‘GLOG_EXPORT’ does not name a type
  111 | extern GLOG_EXPORT int SetVLOGLevel(const char* module_pattern, int log_level);
      |        ^~~~~~~~~~~
/usr/include/glog/vlog_is_on.h:130:8: error: ‘GLOG_EXPORT’ does not name a type
  130 | extern GLOG_EXPORT bool InitVLOG3__(SiteFlag* site_flag,
      |        ^~~~~~~~~~~
/usr/include/glog/logging.h:82:20: error: variable ‘google::GLOG_EXPORT google::LogMessageTime’ has initializer but incomplete type
   82 | struct GLOG_EXPORT LogMessageTime {
      |                    ^~~~~~~~~~~~~~
/usr/include/glog/logging.h:83:19: error: expected ‘}’ before ‘;’ token
   83 |   LogMessageTime();
      |                   ^
/usr/include/glog/logging.h:82:35: note: to match this ‘{’
   82 | struct GLOG_EXPORT LogMessageTime {
      |                                   ^
/usr/include/glog/logging.h:84:3: error: ‘explicit’ outside class declaration
   84 |   explicit LogMessageTime(std::chrono::system_clock::time_point now);
      |   ^~~~~~~~
/usr/include/glog/logging.h:87:13: error: non-member function ‘time_t google::timestamp()’ cannot have cv-qualifier
   87 |       const noexcept {
      |             ^~~~~~~~
/usr/include/glog/logging.h: In function ‘time_t google::timestamp()’:
/usr/include/glog/logging.h:88:49: error: ‘when’ was not declared in this scope
   88 |     return std::chrono::system_clock::to_time_t(when());
      |                                                 ^~~~
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:90:61: error: non-member function ‘const std::chrono::_V2::system_clock::time_point& google::when()’ cannot have cv-qualifier
   90 |   const std::chrono::system_clock::time_point& when() const noexcept {
      |                                                             ^~~~~~~~
/usr/include/glog/logging.h: In function ‘const std::chrono::_V2::system_clock::time_point& google::when()’:
/usr/include/glog/logging.h:91:12: error: ‘timestamp_’ was not declared in this scope; did you mean ‘timestamp’?
   91 |     return timestamp_;
      |            ^~~~~~~~~~
      |            timestamp
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:93:19: error: non-member function ‘int google::sec()’ cannot have cv-qualifier
   93 |   int sec() const noexcept { return tm_.tm_sec; }
      |                   ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::sec()’:
/usr/include/glog/logging.h:93:37: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   93 |   int sec() const noexcept { return tm_.tm_sec; }
      |                                     ^~~
      |                                     tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:94:21: error: non-member function ‘long int google::usec()’ cannot have cv-qualifier
   94 |   long usec() const noexcept { return usecs_.count(); }
      |                     ^~~~~~~~
/usr/include/glog/logging.h: In function ‘long int google::usec()’:
/usr/include/glog/logging.h:94:39: error: ‘usecs_’ was not declared in this scope; did you mean ‘usec’?
   94 |   long usec() const noexcept { return usecs_.count(); }
      |                                       ^~~~~~
      |                                       usec
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:95:20: error: non-member function ‘int google::min()’ cannot have cv-qualifier
   95 |   int(min)() const noexcept { return tm_.tm_min; }
      |                    ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::min()’:
/usr/include/glog/logging.h:95:38: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   95 |   int(min)() const noexcept { return tm_.tm_min; }
      |                                      ^~~
      |                                      tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:96:20: error: non-member function ‘int google::hour()’ cannot have cv-qualifier
   96 |   int hour() const noexcept { return tm_.tm_hour; }
      |                    ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::hour()’:
/usr/include/glog/logging.h:96:38: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   96 |   int hour() const noexcept { return tm_.tm_hour; }
      |                                      ^~~
      |                                      tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:97:19: error: non-member function ‘int google::day()’ cannot have cv-qualifier
   97 |   int day() const noexcept { return tm_.tm_mday; }
      |                   ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::day()’:
/usr/include/glog/logging.h:97:37: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   97 |   int day() const noexcept { return tm_.tm_mday; }
      |                                     ^~~
      |                                     tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:98:21: error: non-member function ‘int google::month()’ cannot have cv-qualifier
   98 |   int month() const noexcept { return tm_.tm_mon; }
      |                     ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::month()’:
/usr/include/glog/logging.h:98:39: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   98 |   int month() const noexcept { return tm_.tm_mon; }
      |                                       ^~~
      |                                       tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:99:20: error: non-member function ‘int google::year()’ cannot have cv-qualifier
   99 |   int year() const noexcept { return tm_.tm_year; }
      |                    ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::year()’:
/usr/include/glog/logging.h:99:38: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
   99 |   int year() const noexcept { return tm_.tm_year; }
      |                                      ^~~
      |                                      tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:100:25: error: non-member function ‘int google::dayOfWeek()’ cannot have cv-qualifier
  100 |   int dayOfWeek() const noexcept { return tm_.tm_wday; }
      |                         ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::dayOfWeek()’:
/usr/include/glog/logging.h:100:43: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
  100 |   int dayOfWeek() const noexcept { return tm_.tm_wday; }
      |                                           ^~~
      |                                           tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:101:25: error: non-member function ‘int google::dayInYear()’ cannot have cv-qualifier
  101 |   int dayInYear() const noexcept { return tm_.tm_yday; }
      |                         ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::dayInYear()’:
/usr/include/glog/logging.h:101:43: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
  101 |   int dayInYear() const noexcept { return tm_.tm_yday; }
      |                                           ^~~
      |                                           tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:102:19: error: non-member function ‘int google::dst()’ cannot have cv-qualifier
  102 |   int dst() const noexcept { return tm_.tm_isdst; }
      |                   ^~~~~~~~
/usr/include/glog/logging.h: In function ‘int google::dst()’:
/usr/include/glog/logging.h:102:37: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
  102 |   int dst() const noexcept { return tm_.tm_isdst; }
      |                                     ^~~
      |                                     tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:104:13: error: non-member function ‘long int google::gmtoff()’ cannot have cv-qualifier
  104 |       const noexcept {
      |             ^~~~~~~~
/usr/include/glog/logging.h: In function ‘long int google::gmtoff()’:
/usr/include/glog/logging.h:105:12: error: ‘gmtoffset_’ was not declared in this scope; did you mean ‘gmtoff’?
  105 |     return gmtoffset_.count();
      |            ^~~~~~~~~~
      |            gmtoff
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:107:42: error: non-member function ‘std::chrono::seconds google::gmtoffset()’ cannot have cv-qualifier
  107 |   std::chrono::seconds gmtoffset() const noexcept { return gmtoffset_; }
      |                                          ^~~~~~~~
/usr/include/glog/logging.h: In function ‘std::chrono::seconds google::gmtoffset()’:
/usr/include/glog/logging.h:107:60: error: ‘gmtoffset_’ was not declared in this scope; did you mean ‘gmtoffset’?
  107 |   std::chrono::seconds gmtoffset() const noexcept { return gmtoffset_; }
      |                                                            ^~~~~~~~~~
      |                                                            gmtoffset
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:108:29: error: non-member function ‘const tm& google::tm()’ cannot have cv-qualifier
  108 |   const std::tm& tm() const noexcept { return tm_; }
      |                             ^~~~~~~~
/usr/include/glog/logging.h: In function ‘const tm& google::tm()’:
/usr/include/glog/logging.h:108:47: error: ‘tm_’ was not declared in this scope; did you mean ‘tm’?
  108 |   const std::tm& tm() const noexcept { return tm_; }
      |                                               ^~~
      |                                               tm
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:110:2: error: expected unqualified-id before ‘private’
  110 |  private:
      |  ^~~~~~~
/usr/include/glog/logging.h:122:33: error: ‘LogMessageTime’ does not name a type; did you mean ‘LogMessageInfo’?
  122 |                           const LogMessageTime& time_)
      |                                 ^~~~~~~~~~~~~~
      |                                 LogMessageInfo
/usr/include/glog/logging.h:133:9: error: ‘LogMessageTime’ does not name a type; did you mean ‘LogMessageInfo’?
  133 |   const LogMessageTime& time;
      |         ^~~~~~~~~~~~~~
      |         LogMessageInfo
/usr/include/glog/logging.h: In constructor ‘LogMessageInfo::LogMessageInfo(const char*, const char*, const int&, std::thread::id, const int&)’:
/usr/include/glog/logging.h:127:9: error: class ‘LogMessageInfo’ does not have any field named ‘time’
  127 |         time(time_) {}
      |         ^~~~
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:136:1: error: expected declaration before ‘}’ token
  136 | }  // namespace google
      | ^
/usr/include/glog/logging.h:483:1: error: two or more data types in declaration of ‘InitGoogleLogging’
  483 | GLOG_EXPORT void InitGoogleLogging(const char* argv0);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:502:55: error: two or more data types in declaration of ‘InitGoogleLogging’
  502 | [[deprecated("Use InstallPrefixFormatter instead.")]] GLOG_EXPORT void
      |                                                       ^~~~~~~~~~~
/usr/include/glog/logging.h:507:1: error: two or more data types in declaration of ‘IsGoogleLoggingInitialized’
  507 | GLOG_EXPORT bool IsGoogleLoggingInitialized();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:510:1: error: two or more data types in declaration of ‘ShutdownGoogleLogging’
  510 | GLOG_EXPORT void ShutdownGoogleLogging();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:521:1: error: two or more data types in declaration of ‘InstallPrefixFormatter’
  521 | GLOG_EXPORT void InstallPrefixFormatter(PrefixFormatterCallback callback,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:527:1: error: expected initializer before ‘InstallFailureFunction’
  527 | InstallFailureFunction(logging_fail_func_t fail_func);
      | ^~~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:531:19: error: two or more data types in declaration of ‘EnableLogCleaner’
  531 |     "instead.")]] GLOG_EXPORT void
      |                   ^~~~~~~~~~~
/usr/include/glog/logging.h:534:1: error: two or more data types in declaration of ‘EnableLogCleaner’
  534 | GLOG_EXPORT void EnableLogCleaner(const std::chrono::minutes& overdue);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:535:1: error: two or more data types in declaration of ‘DisableLogCleaner’
  535 | GLOG_EXPORT void DisableLogCleaner();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:536:1: error: two or more data types in declaration of ‘SetApplicationFingerprint’
  536 | GLOG_EXPORT void SetApplicationFingerprint(const std::string& fingerprint);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:660:1: error: two or more data types in declaration of ‘MakeCheckOpValueString’
  660 | GLOG_EXPORT void MakeCheckOpValueString(std::ostream* os, const char& v);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:662:1: error: two or more data types in declaration of ‘MakeCheckOpValueString’
  662 | GLOG_EXPORT void MakeCheckOpValueString(std::ostream* os, const signed char& v);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:664:1: error: two or more data types in declaration of ‘MakeCheckOpValueString’
  664 | GLOG_EXPORT void MakeCheckOpValueString(std::ostream* os,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:669:1: error: two or more data types in declaration of ‘MakeCheckOpValueString’
  669 | GLOG_EXPORT void MakeCheckOpValueString(std::ostream* os,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:689:19: error: variable ‘google::GLOG_EXPORT google::logging::internal::CheckOpMessageBuilder’ has initializer but incomplete type
  689 | class GLOG_EXPORT CheckOpMessageBuilder {
      |                   ^~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:690:2: error: expected primary-expression before ‘public’
  690 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:690:2: error: expected ‘}’ before ‘public’
/usr/include/glog/logging.h:689:41: note: to match this ‘{’
  689 | class GLOG_EXPORT CheckOpMessageBuilder {
      |                                         ^
/usr/include/glog/logging.h:690:2: note: probably missing a comma or an operator before
  690 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h: In function ‘std::unique_ptr<std::__cxx11::basic_string<char> > google::logging::internal::MakeCheckOpString(const T1&, const T2&, const char*)’:
/usr/include/glog/logging.h:709:24: error: expected ‘;’ before ‘comb’ [-Wtemplate-body]
  709 |   CheckOpMessageBuilder comb(exprtext);
      |                        ^~~~~
      |                        ;
/usr/include/glog/logging.h:710:26: error: ‘comb’ was not declared in this scope; did you mean ‘wctomb’? [-Wtemplate-body]
  710 |   MakeCheckOpValueString(comb.ForVar1(), v1);
      |                          ^~~~
      |                          wctomb
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:830:1: error: invalid declarator before ‘CheckstrcmptrueImpl’
  830 | DECLARE_CHECK_STROP_IMPL(strcmp, true)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:831:1: error: invalid declarator before ‘CheckstrcmpfalseImpl’
  831 | DECLARE_CHECK_STROP_IMPL(strcmp, false)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:832:1: error: invalid declarator before ‘CheckstrcasecmptrueImpl’
  832 | DECLARE_CHECK_STROP_IMPL(strcasecmp, true)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:833:1: error: invalid declarator before ‘CheckstrcasecmpfalseImpl’
  833 | DECLARE_CHECK_STROP_IMPL(strcasecmp, false)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/glog/logging.h:1198:32: error: expected initializer before ‘:’ token
 1198 | class GLOG_EXPORT LogStreamBuf : public std::streambuf {
      |                                ^
/usr/include/glog/logging.h:1215:23: error: aggregate ‘google::logging::internal::GLOG_NO_EXPORT google::logging::internal::LogMessageData’ has incomplete type and cannot be defined
 1215 | struct GLOG_NO_EXPORT LogMessageData;
      |                       ^~~~~~~~~~~~~~
/usr/include/glog/logging.h:1228:19: error: variable ‘google::GLOG_EXPORT google::LogMessage’ has initializer but incomplete type
 1228 | class GLOG_EXPORT LogMessage {
      |                   ^~~~~~~~~~
/usr/include/glog/logging.h:1229:2: error: expected primary-expression before ‘public’
 1229 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1229:2: error: expected ‘}’ before ‘public’
/usr/include/glog/logging.h:1228:30: note: to match this ‘{’
 1228 | class GLOG_EXPORT LogMessage {
      |                              ^
/usr/include/glog/logging.h:1229:2: note: probably missing a comma or an operator before
 1229 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1410:35: error: expected initializer before ‘:’ token
 1410 | class GLOG_EXPORT LogMessageFatal : public LogMessage {
      |                                   ^
/usr/include/glog/logging.h:1434:25: error: expected initializer before ‘&’ token
 1434 | GLOG_EXPORT std::ostream& operator<<(std::ostream& os, const Counter_t&);
      |                         ^
/usr/include/glog/logging.h:1437:35: error: expected initializer before ‘:’ token
 1437 | class GLOG_EXPORT ErrnoLogMessage : public LogMessage {
      |                                   ^
/usr/include/glog/logging.h: In function ‘T google::logging::internal::CheckNotNull(const char*, int, const char*, T&&)’:
/usr/include/glog/logging.h:1468:5: error: there are no arguments to ‘LogMessageFatal’ that depend on a template parameter, so a declaration of ‘LogMessageFatal’ must be available [-Wtemplate-body]
 1468 |     LogMessageFatal(file, line, std::make_unique<std::string>(names));
      |     ^~~~~~~~~~~~~~~
/usr/include/glog/logging.h:1468:5: note: (if you use ‘-fpermissive’, G++ will accept your code, but allowing the use of an undeclared name is deprecated)
/usr/include/glog/logging.h: At global scope:
/usr/include/glog/logging.h:1484:1: error: two or more data types in declaration of ‘FlushLogFiles’
 1484 | GLOG_EXPORT void FlushLogFiles(LogSeverity min_severity);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1489:1: error: two or more data types in declaration of ‘FlushLogFilesUnsafe’
 1489 | GLOG_EXPORT void FlushLogFilesUnsafe(LogSeverity min_severity);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1496:1: error: two or more data types in declaration of ‘SetLogDestination’
 1496 | GLOG_EXPORT void SetLogDestination(LogSeverity severity,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1505:1: error: two or more data types in declaration of ‘SetLogSymlink’
 1505 | GLOG_EXPORT void SetLogSymlink(LogSeverity severity,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1513:19: error: variable ‘google::GLOG_EXPORT google::LogSink’ has initializer but incomplete type
 1513 | class GLOG_EXPORT LogSink {
      |                   ^~~~~~~
/usr/include/glog/logging.h:1514:2: error: expected primary-expression before ‘public’
 1514 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1514:2: error: expected ‘}’ before ‘public’
/usr/include/glog/logging.h:1513:27: note: to match this ‘{’
 1513 | class GLOG_EXPORT LogSink {
      |                           ^
/usr/include/glog/logging.h:1514:2: note: probably missing a comma or an operator before
 1514 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1553:1: error: two or more data types in declaration of ‘AddLogSink’
 1553 | GLOG_EXPORT void AddLogSink(LogSink* destination);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1553:38: error: ‘destination’ was not declared in this scope
 1553 | GLOG_EXPORT void AddLogSink(LogSink* destination);
      |                                      ^~~~~~~~~~~
/usr/include/glog/logging.h:1554:1: error: two or more data types in declaration of ‘RemoveLogSink’
 1554 | GLOG_EXPORT void RemoveLogSink(LogSink* destination);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1554:41: error: ‘destination’ was not declared in this scope
 1554 | GLOG_EXPORT void RemoveLogSink(LogSink* destination);
      |                                         ^~~~~~~~~~~
/usr/include/glog/logging.h:1562:1: error: two or more data types in declaration of ‘SetLogFilenameExtension’
 1562 | GLOG_EXPORT void SetLogFilenameExtension(const char* filename_extension);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1569:1: error: two or more data types in declaration of ‘SetStderrLogging’
 1569 | GLOG_EXPORT void SetStderrLogging(LogSeverity min_severity);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1574:1: error: two or more data types in declaration of ‘LogToStderr’
 1574 | GLOG_EXPORT void LogToStderr();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1582:1: error: two or more data types in declaration of ‘SetEmailLogging’
 1582 | GLOG_EXPORT void SetEmailLogging(LogSeverity min_severity,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1587:1: error: two or more data types in declaration of ‘SendEmail’
 1587 | GLOG_EXPORT bool SendEmail(const char* dest, const char* subject,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1590:43: error: invalid declarator before ‘&’ token
 1590 | GLOG_EXPORT const std::vector<std::string>& GetLoggingDirectories();
      |                                           ^
/usr/include/glog/logging.h:1595:1: error: two or more data types in declaration of ‘ReprintFatalMessage’
 1595 | GLOG_EXPORT void ReprintFatalMessage();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1604:1: error: two or more data types in declaration of ‘TruncateLogFile’
 1604 | GLOG_EXPORT void TruncateLogFile(const char* path, uint64 limit, uint64 keep);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1609:1: error: two or more data types in declaration of ‘TruncateStdoutStderr’
 1609 | GLOG_EXPORT void TruncateStdoutStderr();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1613:1: error: two or more data types in declaration of ‘GetLogSeverityName’
 1613 | GLOG_EXPORT const char* GetLogSeverityName(LogSeverity severity);
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1628:19: error: variable ‘google::GLOG_EXPORT google::base::Logger’ has initializer but incomplete type
 1628 | class GLOG_EXPORT Logger {
      |                   ^~~~~~
/usr/include/glog/logging.h:1629:2: error: expected primary-expression before ‘public’
 1629 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1629:2: error: expected ‘}’ before ‘public’
/usr/include/glog/logging.h:1628:26: note: to match this ‘{’
 1628 | class GLOG_EXPORT Logger {
      |                          ^
/usr/include/glog/logging.h:1629:2: note: probably missing a comma or an operator before
 1629 |  public:
      |  ^~~~~~
/usr/include/glog/logging.h:1662:26: error: expected initializer before ‘*’ token
 1662 | extern GLOG_EXPORT Logger* GetLogger(LogSeverity level);
      |                          ^
/usr/include/glog/logging.h:1667:54: error: ‘Logger’ is not a type
 1667 | extern GLOG_EXPORT void SetLogger(LogSeverity level, Logger* logger);
      |                                                      ^~~~~~
/usr/include/glog/logging.h:1667:8: error: two or more data types in declaration of ‘SetLogger’
 1667 | extern GLOG_EXPORT void SetLogger(LogSeverity level, Logger* logger);
      |        ^~~~~~~~~~~
/usr/include/glog/logging.h:1672:30: error: expected initializer before ‘:’ token
 1672 | class GLOG_EXPORT NullStream : public LogMessage::LogStream {
      |                              ^
/usr/include/glog/logging.h:1697:8: error: ‘NullStream’ does not name a type; did you mean ‘CUstream’?
 1697 | inline NullStream& operator<<(NullStream& str, const T&) {
      |        ^~~~~~~~~~
      |        CUstream
/usr/include/glog/logging.h:1703:35: error: expected initializer before ‘:’ token
 1703 | class GLOG_EXPORT NullStreamFatal : public NullStream {
      |                                   ^
/usr/include/glog/logging.h:1727:1: error: two or more data types in declaration of ‘InstallFailureSignalHandler’
 1727 | GLOG_EXPORT void InstallFailureSignalHandler();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1730:1: error: two or more data types in declaration of ‘IsFailureSignalHandlerInstalled’
 1730 | GLOG_EXPORT bool IsFailureSignalHandlerInstalled();
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1736:1: error: two or more data types in declaration of ‘InstallFailureWriter’
 1736 | GLOG_EXPORT void InstallFailureWriter(void (*writer)(const char* data,
      | ^~~~~~~~~~~
/usr/include/glog/logging.h:1740:25: error: expected initializer before ‘GetStackTrace’
 1740 | GLOG_EXPORT std::string GetStackTrace();
      |                         ^~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
```

**Expected behavior**
A clear and concise description of what you expected to happen.

**ds_report output**
Please run `ds_report` to give us details about your setup.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**System info (please complete the following information):**
 - OS: Linux 6.15.2
 - GPU count and types: 1, NVIDIA GeForce GTX 1660 Ti
 - Interconnects (if applicable) [e.g., two machines connected with 100 Gbps IB]
 - Python version: 3.13.3
 - Any other relevant info about your setup

**Launcher context**
Are you launching your experiment with the `deepspeed` launcher, MPI, or something else?
Yes

**Docker context**
Are you using a specific docker image that you can share?
No

**Additional context**
Add any other context about the problem here.

google-glog version: 0.7.1
cuda version: 12.8.1

--- a/op_builder/builder.py
+++ b/op_builder/builder.py
@@ -571,6 +571,8 @@ class OpBuilder(ABC):
         nvcc_args = self.strip_empty_entries(self.nvcc_args())
         cxx_args = self.strip_empty_entries(self.cxx_args())
 
+        cxx_args.append("-UC10_USE_GLOG")
+        nvcc_args.append("-UC10_USE_GLOG")
         if isinstance(self, CUDAOpBuilder):
             if not self.build_for_cpu and self.enable_bf16:
                 cxx_args.append("-DBF16_AVAILABLE")

